packages/xrpl/src/models/methods/tx.ts

```diff
@@ -44,6 +51,11 @@ export interface TxResponse<T extends BaseTransaction = Transaction>
   result: {
     /** The SHA-512 hash of the transaction. */
     hash: string
+    /**
+     * The Concise Transaction Identifier of the transaction (16-byte hex string)
+     * Not included in responses from reporting mode servers.
```

Comment:
I think it is available in reporting mode but not clio.

Reply:
For now, I think it's in rippled and rippled reporting mode 1.12 and later